File-Formats/PDF/Convert-HTML-To-PDF/azure.md

```diff
@@ -318,133 +301,109 @@ public static async Task<IActionResult> Run([HttpTrigger(AuthorizationLevel.Func
-
-string[] executableFiles = new string[] { "chrome", "chrome_sandbox" };
-
-foreach (string executable in executableFiles)
-{
-    var execPath = Path.Combine(tempBlinkDir, executable);
-
-    if (File.Exists(execPath))
-    {
-        var code = Function1.Chmod(execPath, ExecutableFilePermissions);
-        if (code != 0)
-        {
-            throw new Exception("Chmod operation failed");
-        }
-    }
+   }
+   return tempBlinkDir;
+   }
+   private static void CopyFilesRecursively(string sourcePath, string targetPath)
+   {
+   //Create all the directories from the source to the destination path.
+   foreach (string dirPath in Directory.GetDirectories(sourcePath, "*", SearchOption.AllDirectories))
+   {
+       Directory.CreateDirectory(dirPath.Replace(sourcePath, targetPath));
+   }
+   //Copy all the files from the source path to the destination path.
+   foreach (string newPath in Directory.GetFiles(sourcePath, "*.*", SearchOption.AllDirectories))
+   {
+       File.Copy(newPath, newPath.Replace(sourcePath, targetPath), true);
+   }
+   }
+   [DllImport("libc", SetLastError = true, EntryPoint = "chmod")]
+   internal static extern int Chmod(string path, FileAccessPermissions mode);
+   private static void SetExecutablePermission(string tempBlinkDir)
+   {
+   FileAccessPermissions ExecutableFilePermissions = FileAccessPermissions.UserRead | FileAccessPermissions.UserWrite | FileAccessPermissions.UserExecute |
+   FileAccessPermissions.GroupRead | FileAccessPermissions.GroupExecute | FileAccessPermissions.OtherRead | FileAccessPermissions.OtherExecute;
+   string[] executableFiles = new string[] { "chrome", "chrome_sandbox" };
+   foreach (string executable in executableFiles)
+   {
+       var execPath = Path.Combine(tempBlinkDir, executable);
+       if (File.Exists(execPath))
+       {
+          var code = Function1.Chmod(execPath, ExecutableFilePermissions);
+           if (code != 0)
+           {
+               throw new Exception("Chmod operation failed");
+           }
+       }
     }
-}
 
-{% endhighlight %}
+       {% endhighlight %}
 
-Include the below enum in the Function1.cs file. 
+7. Include the below enum in the Function1.cs file. 
 
-{% highlight c# tabtitle="C#" %}
+   {% highlight c# tabtitle="C#" %}
 
-[Flags]
-internal enum FileAccessPermissions : uint
-{
-OtherExecute = 1,
-OtherWrite = 2,
-OtherRead = 4,
-
-GroupExecute = 8,
-GroupWrite = 16,
-GroupRead = 32,
-
-UserExecute = 64,
-UserWrite = 128,
-UserRead = 256
-}
+   [Flags]
+   internal enum FileAccessPermissions : uint
+   {
+   OtherExecute = 1,
+   OtherWrite = 2,
+   OtherRead = 4,
+   GroupExecute = 8,
+   GroupWrite = 16,
+   GroupRead = 32,
+   UserExecute = 64,
+   UserWrite = 128,
+   UserRead = 256
+   }
 
-{% endhighlight %}
+   {% endhighlight %}
 
-### Publish to Azure Functions Linux
+## Publish to Azure Functions Linux
```

Comment:
This should be third level heading only  - Leave it as like before

Reply:
modified